src/Security/Security/Az.Security.psd1

```diff
@@ -12,7 +12,7 @@
 # RootModule = ''
 
 # Version number of this module.
-ModuleVersion = '0.7.2'
+ModuleVersion = '0.7.3'
```

Comment:
@eliagrady you can revert the versioning changes you made in this PR (`Az.Security.psd1`, `ChangeLog.md`, `AssemblyInfo.cs`) since we will handle that before releasing the module

Reply:
done.